performance/plugins/extrinsic3rdPartySchedule.ts

```diff
@@ -0,0 +1,137 @@
+  async arrange(testParams: TestParams): Promise<boolean> {
+    await initApi(testParams.nodes[0]);
+    getApi();
+    setupUsers();
+    await setupApi();
+    await super.arrange(testParams);
+    //create tokens for testing.
+    const mgaNode = new Node(testParams.nodes[0]);
+    await mgaNode.connect();
+    const keyring = new Keyring({ type: "sr25519" });
+    const sudo = UserFactory.createUser(Users.SudoUser, keyring, mgaNode);
+
+    await initApi(testParams.nodes[0]);
+    this.tokens = await Assets.setupUserWithCurrencies(
+      sudo,
+      [
+        new BN("1000000000000000000000000000"),
+        new BN("1000000000000000000000000000"),
+      ],
+      sudo,
+    ).then((values) => {
+      return values.map((val) => val.toNumber());
+    });
+    await Sudo.batchAsSudoFinalized(
+      Assets.mintNative(sudo, Assets.DEFAULT_AMOUNT.muln(40e6)),
+      Assets.mintNative(sudo, Assets.DEFAULT_AMOUNT.muln(40e6)),
+      Assets.mintNative(sudo, Assets.DEFAULT_AMOUNT.muln(40e6)),
+      Assets.mintNative(sudo, Assets.DEFAULT_AMOUNT.muln(40e6)),
+      Assets.mintNative(sudo, Assets.DEFAULT_AMOUNT.muln(40e6)),
+      Assets.mintNative(sudo, Assets.DEFAULT_AMOUNT.muln(40e6)),
+      Assets.mintNative(sudo, Assets.DEFAULT_AMOUNT.muln(40e6)),
+      Assets.mintNative(sudo, Assets.DEFAULT_AMOUNT.muln(40e6)),
+      Assets.mintNative(sudo, Assets.DEFAULT_AMOUNT.muln(40e6)),
+      Assets.mintToken(
+        new BN(this.tokens[0]),
+        sudo,
+        Assets.DEFAULT_AMOUNT.muln(40e6),
+      ),
+      Assets.mintToken(
+        new BN(this.tokens[1]),
+        sudo,
+        Assets.DEFAULT_AMOUNT.muln(40e6),
+      ),
+      Xyk.createPool(
+        MGA_ASSET_ID,
+        Assets.DEFAULT_AMOUNT.muln(10e6),
+        new BN(this.tokens[0]),
+        Assets.DEFAULT_AMOUNT.muln(10e6),
+      ),
+      Xyk.createPool(
+        MGA_ASSET_ID,
+        Assets.DEFAULT_AMOUNT.muln(10e6),
+        new BN(this.tokens[1]),
+        Assets.DEFAULT_AMOUNT.muln(10e6),
+      ),
+    );
+
+    await this.mintTokensToUsers(testParams.threads, testParams.nodes);
+    // await this.mintERC20TokensToUsers(this.tokens, this.mgaNodeandUsers);
+    console.info(`Setup Done!`);
+    tokens = this.tokens;
+    return true;
+  }
+  async act(testParams: TestParams): Promise<boolean> {
+    await super.act(testParams);
+    //    const generator = (
+    //      users: { nonce: BN; keyPair: KeyringPair }[],
+    //      thread: number,
+    //      offset: BN,
+    //    ) => {
+    //      return createAndSignNewSchedules(users, thread, offset);
+    //    };
+    await runTransactions(
+      this.mgaNodeandUsers,
+      testParams,
+      createAndSignNewSchedules,
+    );
+    console.info(`.... Done Sending Txs`);
+    return true;
+  }
+}
+
+async function createAndSignNewSchedules(
+  mgaSdk: MangataInstance,
+  users: { nonce: BN; keyPair: KeyringPair }[],
+  threadId: number,
+  nonceOffset: BN = new BN(0),
+) {
+  await mgaSdk.api();
```

Comment:
It's the same as [here](https://github.com/mangata-finance/mangata-e2e/pull/354/files#r1426484180)